sunpy/map/tests/test_mapbase.py

```diff
@@ -1036,24 +1036,21 @@ def test_rotate_with_incompatible_missing_dtype_warning():
                      observer='earth', frame=sunpy.coordinates.Helioprojective)
     header = sunpy.map.make_fitswcs_header(data, coord)
     test_map = sunpy.map.Map(data, header)
+    
     with pytest.warns(SunpyUserWarning,
                       match="The specified `missing` value is not an integer, but the data "
-                      "array is of integer type, so the output may be strange."):
+                            "array is of integer type, so the output may be strange."):
         test_map.rotate(order=3, missing=np.nan)
 
-
-@pytest.mark.skipif(version.parse(np.__version__) <= version.parse("1.20.0"),
```

Comment:
Can you remove this on the other test as well?

Reply:
can you please elaborate what you want me to remove?

Reply:
There were two tests marked with a pytest skipif. This should be removed

Reply:
I think there is only one test_mapbase.py with skipif one is with xfail.

Reply:
Is there not another one above this test?

Reply:
the one above it is the xfail

Reply:
Ah I didn't notice. Either way that should removed and the test fixed if needed.